entrypoint.sh

```diff
@@ -1,5 +1,28 @@
-#!/bin/sh
+#!/bin/bash
 
+done=0
+
+function signals()
+{
+  if [ $done -eq 0 ]; then
+    kill $(pgrep vme)
+    kill $(pgrep mplex)
+  fi
+  done=1
+}
+
+trap signals INT TERM EXIT HUP
+
+service nginx restart
```

Comment:
Is entrypoint only for docker? If not, isn't it kind of risky to restart nginx?

Reply:
Docker only